_posts/2021-06-28-android-audio-player-tutorial-with-armadillo.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "Armadillo makes audio players in Android easy"
+tags:
+- android
+- kotlin
+- armadillo
+- featured
+author: nathans
+team: Android
+---
+
+Armadillo is the fully featured audio player library Scribd uses to play and
+download all of its audiobooks and podcasts, which is [now open
+source](https://github.com/scribd/armadillo). It specializes in playing HLS
+or MP3 content that is broken down into chapters or tracks. It leverages
+[Google’s Exoplayer](https://github.com/google/ExoPlayer/) library for its audio engine. Exoplayer wraps a variety of
+low level audio and video apis but has few opinions of its own for actually
+using audio in an Android app.
+
+![Armadillo Image](https://i.ibb.co/LzTK79B/armadillo.jpg)
+<sup>Icon made by [Freepik](https://www.flaticon.com/authors/freepik) from [www.flaticon.com](http://www.flaticon.com/) <sub>
```

Comment:
We should not use an icon if we have to attribute it like this. If this is mandatory, then we need a new image

Reply:
Icon has been removed and updated with one that we own.